Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/ForwardPassOutput.azsli

```diff
@@ -27,7 +27,7 @@ struct ForwardPassOutput
     float4 m_specularColor : SV_Target1;     //!< RGB = Specular Lighting, A = Unused
     float4 m_albedo : SV_Target2;            //!< RGB = Surface albedo pre-multiplied by other factors that will be multiplied later by diffuse GI, A = specularOcclusion
     float4 m_specularF0 : SV_Target3;        //!< RGB = Specular F0, A = roughness
-    float4 m_normal : SV_Target4;            //!< RGB10 = EncodeNormalSignedOctahedron(worldNormal), A2 = multiScatterCompensationEnabled
+    float4 m_normal : SV_Target4;            //!< RGB10 = EncodeNormalSignedOctahedron(worldNormal), A2 = Flags
```

Comment:
Can we leave a helpful breadcrumb instead of just "Flags"? Following the pattern already here, maybe change to this:
```suggestion
    float4 m_normal : SV_Target4;            //!< RGB10 = EncodeNormalSignedOctahedron(worldNormal), A2 = EncodeUnorm2BitFlags(o_enableIBL, o_specularF0_enableMultiScatterCompensation)
```

Reply:
I can add a little here, but I think there's value in encouraging code inspection on these types of RTs.